app/src/main/java/com/babylon/wallet/android/di/DataModule.kt

```diff
@@ -107,4 +109,10 @@ interface DataModule {
     fun bindPreferenceManager(
         preferencesManager: PreferencesManagerImpl
     ): PreferencesManager
+
+    @Binds
+    @Singleton
+    fun bindAppEventBus(
+        appEventBus: AppEventBusImpl
+    ): AppEventBus
```

Comment:
you don't really need this. I see that you created a `FakeAppEventBus` but you don't really use it. 
In the `CreatePersonaViewModelTest` you can mock the `AppEventBus` like we already did to the rest of the viewmodel tests. 

Although I would prefer to have fakes, especially when we really need to produce some fake values, but currently we don't do that with the `AppEventBus`.

Reply:
if code under the test calls event bus mock, I need to mock `send` method, otherwise test will fail. With Fake, although it's implementation does nothing, I don't need to do `every { bus.send(any()) } just Runs`

Reply:
ok if you keep this binding then you should remove the annotations `@Singleton` and `@Inject` from the actual implementation of `AppEventBusImpl`